rep-2000.rst

```diff
@@ -187,10 +190,13 @@ Significant errors discovered in Tier 1 platforms can impact release dates and w
 
 Tier 2
 
-Tier 2 platforms are subject to ad hoc testing and binary packages may not be provided or supported.
+Tier 2 platforms are subject to periodic CI testing which runs both test builds and unit tests with publically accessible results.
```

Comment:
I find the phrase `test builds` a bit confusing and don't think the tests are limited to unit tests. HOw about updating this to:

> ... runs both builds and tests with ...